docs/user-guides/4-creating-for-webaverse.md

```diff
@@ -0,0 +1,18 @@
+---
+id: creating-for-webaverse
+title: Creating for the Webaverse
+---
+
+If you've read the previous sections of the User Guides, it should be clear that the Webaverse simply revolves around "applications" packaged into "XRPackages" (XRPKs).
+
+You can just drop the XRPackages into a world, and what you make is only limited by your imagination.
+
+**We've made it incredibly easy to get started developing for the Webaverse.**
```

Comment:
I've made this comment elsewhere, but these kinds of "salesy" mentions are probably best left out of the documentation, whose purpose is primarily to inform and not to convince :D.